src/client/app/components/HomeComponent.jsx

```diff
@@ -15,6 +17,7 @@ export default function HomeComponent() {
 		<div>
 			<HeaderComponent renderLoginButton />
 			<DashboardContainer />
+			<ExportContainer />
```

Comment:
Could you move this to be in UIOptionsComponent (at the bottom) instead of here?